app/views/admin/loans/_transactions.html.slim

```diff
@@ -0,0 +1,14 @@
+section.transactions
+  .show-actions.hidden-print
+    a.btn.btn-primary data-action="new-transaction" href="#" = t(".new_transaction")
+
+  = grid(@transactions_grid) do |g|
+    - g.column name: 'Id', attribute: 'qb_id'
+    - g.column name: 'Type' do |transaction|
+      - transaction.qb_transaction_type
+    - g.column name: 'Date' do |transaction|
+      - transaction.txn_date
+    - g.column name: 'Total' do |transaction|
+      - transaction.total
+    - g.column name: 'Note' do |transaction|
```

Comment:
These grid headers should be i18n'd. Preferably using the Wice default lookup location instead of calling `t` for each one, if possible.